packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalCardUnconfirmed.tsx

```diff
@@ -1,24 +1,34 @@
-import { getNetworkName } from '../../util/networks'
+import { getNetworkName, isNetwork } from '../../util/networks'
 import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
 import { MergedTransaction } from '../../state/app/state'
 import { WithdrawalCountdown } from '../common/WithdrawalCountdown'
 import { WithdrawalCardContainer, WithdrawalL2TxStatus } from './WithdrawalCard'
 import { Button } from '../common/Button'
 import { Tooltip } from '../common/Tooltip'
 import { formatAmount } from '../../util/NumberUtils'
+import { useMemo } from 'react'
+import { sanitizeTokenSymbol } from '../../util/TokenUtils'
 
 export function WithdrawalCardUnconfirmed({ tx }: { tx: MergedTransaction }) {
-  const { l1 } = useNetworksAndSigners()
+  const { l1, l2 } = useNetworksAndSigners()
   const networkName = getNetworkName(l1.network.id)
 
+  const tokenSymbol = useMemo(
+    () =>
+      sanitizeTokenSymbol(tx.asset.toUpperCase(), {
+        erc20L1Address: tx.tokenAddress || '',
```

Comment:
We can have the functions accept `erc20L1Address` as `string | undefined` and we just return the symbol in case it's undefined. Will make the call sites a bit cleaner

Reply:
Done